mkdocs.yml

```diff
@@ -25,18 +25,20 @@ pages:
   - 'Worker Node (OASIS install)': 'worker-node/install-wn-oasis.md'
   - 'CVMFS': 'worker-node/install-cvmfs.md'
   - 'Singularity': 'worker-node/install-singularity.md'
-- Data:
+- Data - HTTP & CVMFS:
```

Comment:
I wish we could have working sub-headings in the ToC but I haven't had any luck getting them to work

Reply:
Yeah - this was good enough for me... there's a mkdocs open PR about this that I sometimes am tempted to try out.